irohad/model/CMakeLists.txt

```diff
@@ -3,15 +3,26 @@ add_subdirectory(converters)
 
 add_library(model
     model_crypto_provider_impl.cpp
-    model_hash_provider_impl.cpp
     impl/model_operators.cpp
+    converters/impl/pb_block_factory.cpp
+    converters/impl/pb_transaction_factory.cpp
+    converters/impl/pb_command_factory.cpp
```

Comment:
Looks like something went wrong, since these files are in `*_model_converters` targets in `irohad/model/converters/CMakeLists.txt`.

Reply:
Thanks, fixed.